walkthroughs/week-3-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java

```diff
@@ -48,6 +48,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Respond with the result.
     response.setContentType("text/html;");
     response.getWriter().println(Arrays.toString(words));
+    response.sendRedirect("https://google.com");
```

Comment:
Is this to test out redirects?

Reply:
Yes, this was one of the steps in the step 3 tutorial to add this into the example and test out a redirect to another website.